src/simplify_reshapes.cpp

```diff
@@ -806,6 +806,31 @@ struct find_transpose_slice
     }
 };
 
+// replace scalar (multibroadcast->unsqueeze) with just an updated (multibroadcast)
+struct find_scalar_multibroadcast_unsqueeze
+{
+    auto matcher() const
+    {
+        auto contiguous = match::name("contiguous");
+        auto mbr        = match::name("multibroadcast");
+        return match::name("unsqueeze")(match::arg(0)(match::skip(contiguous)(mbr.bind("mbr"))));
```

Comment:
This should be `match::name("flatten", "reshape", "squeeze", "unsqueeze", "transpose")` as this transformation works for all those operators as well.

Reply:
Done